plugins/performance-lab/includes/site-health/audit-enqueued-assets/helper.php

```diff
@@ -249,10 +249,10 @@ function perflab_aea_get_total_size_bytes_enqueued_scripts() {
  * @return int|false Number of total styles or false if transient hasn't been set.
  */
 function perflab_aea_get_total_enqueued_styles() {
-	$enqueued_styles      = false;
-	$list_enqueued_styles = get_transient( 'aea_enqueued_front_page_styles' );
-	if ( is_array( $list_enqueued_styles ) ) {
-		$enqueued_styles = count( $list_enqueued_styles );
+	$enqueued_styles = false;
+	$blocking_assets = get_transient( 'aea_blocking_assets' );
+	if ( is_array( $blocking_assets ) && isset( $blocking_assets['styles'] ) && is_array( $blocking_assets['styles'] ) ) {
```

Comment:
Minor suggestion, I believe this can be simplified to:

```suggestion
	if ( isset( $blocking_assets['styles'] ) && is_array( $blocking_assets['styles'] ) ) {
```